net/src/ipv6/mod.rs

```diff
@@ -23,13 +23,19 @@ use tracing::{debug, trace};
 pub mod addr;
 pub mod flow_label;
 
+#[allow(unused_imports)] // re-export
+#[cfg(any(test, feature = "arbitrary"))]
+pub use contract::*;
```

Comment:
I have no idea where this is coming from or how this is related to the rest of the commit.

Reply:
this is a rebase mistake.  Will fix